modules/libcom/src/misc/epicsString.h

```diff
@@ -82,4 +85,13 @@ LIBCOM_API int dbTranslateEscape(char *s, const char *ct);
 }
 #endif
 
+#if __cplusplus >= 201103L
+
+template<size_t N>
+static inline char* epicsStrnCpy(char(&dest)[N], const char* src) {
+    return epicsStrnCpy(dest, src, N);
+}
```

Comment:
This seems like it might incentivize bad code (C string manipulation in C++).

Also, it should be used somewhere in Base.  This is especially important for a C++ template to prove that it can actually be applied.

Reply:
Agreed that it's a good idea to use it in base to ensure it works (if we're going to keep this function that is!)

I'm not sure it incentivizes bad code though. `std::string` allocates for sufficiently large strings and you might want to use traditional C strings (or `std::array`) to avoid that.